drip/views.py

```diff
@@ -1 +1,44 @@
-# Create your views here.
+from typing import Any, Dict, List
+
+from django.views.generic import TemplateView
+
+from drip.tokens import EmailToken
+
+
+class UnsubscribeDripView(TemplateView):
+    template_name = "unsubscribe_drip.html"
+    invalid_template_name = "unsubscribe_drip_invalid.html"
+    success_template_name = "unsubscribe_drip_success.html"
+
+    def _set_user_and_drip(self, **kwargs):
+        drip_uidb64 = kwargs.get("drip_uidb64", "")
+        uidb64 = kwargs.get("uidb64", "")
+        token = kwargs.get("token", "")
+        self.user = EmailToken.validate_user_uidb64_token(uidb64, token)
+        self.drip = EmailToken.validate_drip_uidb64(drip_uidb64)
+        self.post_sucess = False
+
+    def dispatch(self, *args, **kwargs):
```

Comment:
You have to set `Any` type for the args and kwargs right?

Reply:
I apply this suggestion. Thanks!